Headers/DebugServer2/Utils/Log.h

```diff
@@ -50,15 +50,15 @@ void Log(int level, char const *classname, char const *funcname,
 
 #define PRI_PTR_CAST(VAL) ((uintptr_t)(VAL))
 
-#if defined(__DS2_LOG_CLASS_NAME__)
-#define DS2LOG(LVL, ...)                                                       \
-  ds2::Log(ds2::kLogLevel##LVL, __DS2_LOG_CLASS_NAME__, __FUNCTION__,          \
-           __VA_ARGS__)
-#else
-#define DS2LOG(LVL, ...)                                                       \
-  ds2::Log(ds2::kLogLevel##LVL, nullptr, __FUNCTION__, __VA_ARGS__)
+#if defined(COMPILER_MSVC)
+#define FUNCTION_NAME __FUNCTION__
+#elif defined(COMPILER_CLANG) || defined(COMPILER_GCC)
+#define FUNCTION_NAME __PRETTY_FUNCTION__
```

Comment:
You probably want a final thing else here. Something like:
```
#else
#error "Unsupported Compiler"
#endif
```

Reply:
Good point. There are examples of that elsewhere in the code, Look up the message we use and use the same probably.

(Although, to be fair, even with no "else" case, the build would fail because of `FUNCTION_NAME` being undefined.)